src/dvclive/huggingface.py

```diff
@@ -45,7 +45,11 @@ def on_log(
         **kwargs,
     ):
         logs = kwargs["logs"]
+        logged: dict[str, str | float] = {}
         for key, value in logs.items():
+            if key in logged and logged[key] == value:
```

Comment:
[F] This is papering over a crack in the integration where we are basically logging the same metric twice per step.

See https://github.com/iterative/dvclive/actions/runs/7866535786/job/21460950002 / https://github.com/iterative/dvclive/actions/runs/7866443198/job/21460718675 for examples of this.

There may be a more "correct" way to fix the integration but I do not know the transformers library enough to dig into this more.